recipes/libsodium/all/readme-update-cci.txt

```diff
@@ -0,0 +1,9 @@
+2022-05-01
```

Comment:
Not required. It can be removed.

Reply:
I would disagree, this comment explains why the commit could not be found on the master branch...
Which is exactly what you were talking about in the next comment ...

Reply:
We don't add any extra file for documentation in CCI. You can add a single comment in conandata with stable branch line. It's already documented when a project does not release/tag, we use cci.date.

Reply:
How's that?